woocommerce/woocommerce
# Update WC Blocks hooks docs to point to new WC monorepo location

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Changes the URLs in `docs/cart-and-checkout-blocks/hooks/migrated-hooks.md` to point to the monorepo location



### How to test the changes in this Pull Request:

Check all URLs in `docs/cart-and-checkout-blocks/hooks/migrated-hooks.md` and ensure they link to a live page and the correct hook.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
